seacrowd/sea_datasets/mysentence/mysentence.py

```diff
@@ -0,0 +1,149 @@
+"""
+
+_DATASETNAME = "mysentence"
+_DESCRIPTION = """\
+mySentence is a corpus with a total size of around 55K for Myanmar sentence segmentation. In formal Burmese (Myanmar language), sentences are grammatically structured
+and typically end with the "။" pote-ma symbol. However, informal language, more commonly used in daily conversations due to its natural flow, does not always follow predefined
+rules for ending sentences, making it challenging for machines to identify sentence boundaries. In this corpus, each token of the sentences and paragraphs is tagged from start to finish.
+"""
+
+_HOMEPAGE = "https://github.com/ye-kyaw-thu/mySentence"
+_LANGUAGES = ["mya"]
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
+_LOCAL = False
+_URLS = {
+    "sent": {
+        "train": "https://raw.githubusercontent.com/ye-kyaw-thu/mySentence/main/ver1.0/data/data-sent/sent_tagged/train.tagged",
+        "valid": "https://raw.githubusercontent.com/ye-kyaw-thu/mySentence/main/ver1.0/data/data-sent/sent_tagged/valid.tagged",
+        "test": "https://raw.githubusercontent.com/ye-kyaw-thu/mySentence/main/ver1.0/data/data-sent/sent_tagged/test.tagged",
+    },
+    "sent+para": {
+        "train": "https://raw.githubusercontent.com/ye-kyaw-thu/mySentence/main/ver1.0/data/data-sent+para/sent+para_tagged/train.tagged",
+        "valid": "https://raw.githubusercontent.com/ye-kyaw-thu/mySentence/main/ver1.0/data/data-sent+para/sent+para_tagged/valid.tagged",
+        "test": "https://raw.githubusercontent.com/ye-kyaw-thu/mySentence/main/ver1.0/data/data-sent+para/sent+para_tagged/test.tagged",
+    },
+}
+
+_SUPPORTED_TASKS = [Tasks.POS_TAGGING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class MysentenceDataset(datasets.GeneratorBasedBuilder):
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=_DESCRIPTION,
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_seq_label",
+            version=SEACROWD_VERSION,
+            description="sentences SEACrowd schema",
+            schema="seacrowd_seq_label",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}+paragraphs_source",
+            version=SOURCE_VERSION,
+            description="sentences para source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}+paragraphs",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}+paragraphs_seacrowd_seq_label",
+            version=SEACROWD_VERSION,
+            description="sentence para SEACrowd schema",
+            schema="seacrowd_seq_label",
+            subset_id=f"{_DATASETNAME}+paragraphs",
```

Comment:
Hi, I'd argue to refrain using non-words (alphanumeric & underscore) for config naming, also for the `subset_id`
```suggestion
            name=f"{_DATASETNAME}_and_paragraphs_source",
            version=SOURCE_VERSION,
            description="sentences para source schema",
            schema="source",
            subset_id=f"{_DATASETNAME}_and_paragraphs",
        ),
        SEACrowdConfig(
            name=f"{_DATASETNAME}_and_paragraphs_seacrowd_seq_label",
            version=SEACROWD_VERSION,
            description="sentence para SEACrowd schema",
            schema="seacrowd_seq_label",
            subset_id=f"{_DATASETNAME}_and_paragraphs",
```